.github/workflows/lint-codeql.yaml

```diff
@@ -4,6 +4,7 @@ on:
   pull_request:
     branches:
       - v1.10
+      - ft/v1.10/**
   push:
     branches:
       - v1.10
```

Comment:
@jibi :wave: I'm going over the v1.11 backport of the same patch, and was wondering whether we need to add a line under branches here as well?

i.e., 
```
    branches:
        - v1.10
        - ft/v1.10/**
```

in the case of 1.10.

Reply:
Yes, for v1.11 we'll need to change that to `ft/v1.11/**`